core/src/main/java/org/elasticsearch/search/aggregations/metrics/ValuesSourceMetricsAggregationBuilder.java

```diff
@@ -85,6 +94,12 @@ public B param(String name, Object value) {
         return (B) this;
     }
 
+    @SuppressWarnings("unchecked")
+    public B format(String format) {
+        this.format = format;
+        return (B) this;
+    }
```

Comment:
Can you add javadocs to these methods, and in particular for the deprecated ones describe what should be used instead?